src/detox/__init__.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import absolute_import, unicode_literals
+
+from tox import cmdline
+from tox.config import parallel
+
+
+def run():
+    parallel.DEFAULT_PARALLEL = "all"
```

Comment:
Perhaps the default should be `auto`? I just run `detox` on `pytest` and it almost froze my computer when it launched 17 environments at the same time.

Reply:
😅 oh my, yeah it maxes out CPU quite nicely. For tox itself all works ok, but probably because it's dominated by network and disk operations the test suite. Will ammend.

Reply:
On the other side I'm usually running mostly something as ``detox -e py27,py37,fix_lint, coverage``.